constants.go

```diff
@@ -759,6 +759,10 @@ const (
 	// PresetListAccessRequestResourcesRoleName is a name of a preset role that
 	// includes permissions to read access request resources.
 	PresetListAccessRequestResourcesRoleName = "list-access-request-resources"
+
+	// PresetMCPAccessRoleName is a name of a preset role that allows
+	// accessing MCP servers.
+	PresetMCPAccessRoleName = "mcp-access"
```

Comment:
It isn't clear if this role is meant for admins or users. I much prefer the roles which give a name to the "actor", such as "reviewer" or "okta-requester". In that spirit I'd suggest the name "mcp-user".